www/src/components/graphiql-embed.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useRef, useCallback, useLayoutEffect } from "react"
+
+const VIDEO_RATIO = 2 / 3
+
+const GraphiQLEmbed = ({ title, url, query }) => {
+  const encodedQuery = encodeURIComponent(query)
+
+  const [iframeWidth, setIframeWidth] = useState(0)
+  const iframeRef = useRef()
+
+  const handleResize = useCallback(
+    () => setIframeWidth(iframeRef.current.clientWidth),
+    [iframeRef.current]
+  )
+
+  useLayoutEffect(() => {
+    handleResize()
+    window.addEventListener(`resize`, handleResize)
+
+    return () => {
+      window.removeEventListener(`resize`, handleResize)
+    }
+  }, [iframeRef.current])
+
+  return (
+    <iframe
+      ref={iframeRef}
+      className="graphiql-embed"
```

Comment:
Could we just use the `sx` prop here for styling instead of adding a global style?

Reply:
Sure.